patches/jest-haste-map+26.1.0.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/node_modules/jest-haste-map/build/crawlers/node.js b/node_modules/jest-haste-map/build/crawlers/node.js
+index 6b6e11a..2ff08fa 100644
+--- a/node_modules/jest-haste-map/build/crawlers/node.js
++++ b/node_modules/jest-haste-map/build/crawlers/node.js
+@@ -205,7 +205,11 @@ function find(roots, extensions, ignore, callback) {
+ 
+ function findNative(roots, extensions, ignore, callback) {
+   const args = Array.from(roots);
```

Comment:
Wouldn't this be more readable?

`const args = [...Array.from(roots), '(', '-type', 'f', '-o', '-type', 'l', ')'];`